docs/rst/tutorials/core/qos/persistency/persistency.rst

```diff
@@ -125,27 +129,31 @@ when a publisher crashes.
 
     .. code-block:: bash
 
-        1$ export FASTRTPS_DEFAULT_PROFILES_FILE=/home/tutorial/writer_config.xml
-        1$ ros2 topic pub --times 5 --qos-durability system_default /persistency_test std_msgs/String "{data: 'Hello'}"
+        # Terminal 1
+        export FASTRTPS_DEFAULT_PROFILES_FILE=/home/tutorial/writer_config.xml
+        ros2 topic pub --times 5 --qos-durability system_default /persistency_test std_msgs/String "{data: 'Hello'}"
 
-        2$ export FASTRTPS_DEFAULT_PROFILES_FILE=/home/tutorial/reader_config.xml
-        2$ ros2 topic echo --qos-durability system_default /persistency_test std_msgs/String
+        # Terminal 2
+        export FASTRTPS_DEFAULT_PROFILES_FILE=/home/tutorial/reader_config.xml
+        ros2 topic echo --qos-durability system_default /persistency_test std_msgs/String
```

Comment:
Consider including `docker exec -ti persistence_testing /vulcanexus_entrypoint.sh /bin/bash` command first. Same reasoning as above.

If it is accepted, we could replicate it to the subscriber example.

Reply:
idem